examples/tests/with-docker/Earthfile

```diff
@@ -70,3 +71,13 @@ one-target-many-names:
             echo "-----" && \
             docker run c:latest uname -a
     END
+
+if-after:
+    FROM earthly/dind:alpine
+    WITH DOCKER --load a:latest=+multi-from-one
+        RUN docker run a:latest
+    END
+
+    IF [ "true" = "true" ]
+        RUN true
+    END
```

Comment:
This reliably reproduces it locally on older Earthly.

Reply:
nice repro-case!